package.json

```diff
@@ -13,27 +13,25 @@
     "karma-chrome-launcher": "^2.2.0",
     "karma-coverage": "^1.1.1",
     "karma-edge-launcher": "^0.4.2",
-    "karma-firefox-launcher": "^1.0.0",
+    "karma-firefox-launcher": "^1.1.0",
     "karma-ie-launcher": "^1.0.0",
     "karma-jasmine": "^1.1.0",
     "karma-jasmine-html-reporter": "^0.2.2",
     "karma-jasmine-spec-tags": "^1.0.1",
-    "karma-phantomjs-launcher": "^1.0.2",
     "karma-verbose-reporter": "^0.0.6",
+    "puppeteer": "^1.1.1",
     "requirejs": "^2.3.5",
     "shx": "^0.2.2",
     "whatwg-fetch": "^2.0.3",
     "xhr2": "^0.1.4"
   },
   "scripts": {
-    "test": "npm run lint && npm run karma -- --browsers Chrome",
+    "test": "karma start --browsers ChromeHeadlessNoSandbox,FirefoxHeadless",
+    "test-dev": "karma start --browsers ChromeHeadlessNoSandbox --skip-tags Slow",
+    "test-manual": "karma start --no-single-run",
+    "test-win": "karma start --browsers Chrome,Firefox,Edge && karma start --browsers IE --skip-tags FailsIE",
+    "test-coverage": "karma start karma.coverage.conf.js",
     "lint": "eslint .",
```

Comment:
I wonder if we should run "cpplint" also as part of the lint script.

Reply:
The npm scripts are just an abstraction around the js tooling (in-case we change linters, etc). If anything `make lint` should maybe call `npm run lint` and CPPLint. That would be a separate change.